test/dids/OracleResolveDriver.test.ts

```diff
@@ -7,16 +7,113 @@ declare function assert(value: unknown): asserts value;
 /**
  * File to test methods in oracle Resolve Driver class
  * @author Michael Kaufman
- * Date Last Modified: Apr 10, 2024
+ * Date Last Modified: Apr 14, 2024
  */
 
 
 
 //fields for mock resoution calls
-var mockDidText = "mockDid";
-var mockQueryText = "mockDirectoryForQuery";
-var mockDid = "did:oracle:test";
-var mockDirectory = "TEST";
+const mockDidText = "mockDid";
+const mockQueryText = "mockDirectoryForQuery";
+const mockDid = "did:oracle:test";
+const mockDirectory = "TEST";
+const didDocumentInstance = new DidDocument({
```

Comment:
Can this not be in its own file and imported? I think it's common to have one file which is like validDID.json and then invalidDID.json. If you want to test pushing in individual components with TS maybe have a separate module for that too like valid_constructed_DID.ts. All of these could be in a test/inputs or ref or something like that.